Source/carthage/Outdated.swift

```diff
@@ -7,6 +7,30 @@ import Curry
 
 /// Type that encapsulates the configuration and evaluation of the `outdated` subcommand.
 public struct OutdatedCommand: CommandProtocol {
+	enum UpdateAvailabilityAndApplicability: String {
+		case updatesAvailableAllApplicable =
+		"Will be updated to the newest version."
+		case updatesAvailableSomeApplicable =
+		"Will be updated, but not to the newest version because of the specified version in Cartfile."
+		case updatesAvailableNoneApplicable =
+		"Will not be updated because of the specified version in Cartfile."
+		case noUpdatesAvailable =
+		"Will not be updated: no updates available."
```

Comment:
I don't think these strings should be specified here. Using a `String`-based enum only makes sense when using the `RawRepresentable` APIs.

Instead, these strings should be moved to a computed property.